Userland/Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -1335,9 +1333,14 @@ void StyleComputer::collect_animation_into(JS::NonnullGCPtr<Animations::Keyframe
         auto start = resolved_start_property.release_nonnull();
         auto end = resolved_end_property.release_nonnull();
 
-        auto next_value = interpolate_property(*effect->target(), it.key, *start, *end, progress_in_keyframe);
-        dbgln_if(LIBWEB_CSS_ANIMATION_DEBUG, "Interpolated value for property {} at {}: {} -> {} = {}", string_from_property_id(it.key), progress_in_keyframe, start->to_string(), end->to_string(), next_value->to_string());
-        style_properties.set_property(it.key, next_value);
+        if (auto next_value = interpolate_property(*effect->target(), it.key, *start, *end, progress_in_keyframe)) {
+            dbgln_if(LIBWEB_CSS_ANIMATION_DEBUG, "Interpolated value for property {} at {}: {} -> {} = {}", string_from_property_id(it.key), progress_in_keyframe, start->to_string(), end->to_string(), next_value->to_string());
+            style_properties.set_property(it.key, *next_value);
+        } else {
+            // If interpolate_property() fails, the element should not be rendered
+            dbgln_if(LIBWEB_CSS_ANIMATION_DEBUG, "Interpolated value for property {} at {}: {} -> {} is invalid", string_from_property_id(it.key), progress_in_keyframe, start->to_string(), end->to_string());
+            style_properties.set_property(PropertyID::Visibility, IdentifierStyleValue::create(ValueID::Hidden));
```

Comment:
Is this a sensible way to accomplish "not rendering" the element? It feels odd, but I wasn't sure how else to do it.

Reply:
I'm not entirely sure what's the best strategy here.
Overriding the `visibility` property is probably not entirely correct since it's observable from JavaScript, but we can do it this way for now, and then figure out something different after we look at how other browsers behave :)